python/triton/experimental/gluon/language/amd/_layouts.py

```diff
@@ -20,10 +20,10 @@ class AMDMFMALayout(DistributedLayout):
     Args:
         version (int): Major and minor identifier for the MFMA instruction.
         instr_shape: (M, N) dimension for the instrinsic shape.
-        transposed: indicates the result tensor is transposed so that each thread holds consecutive elements in the same row instead of column, which is good for chained dot and global write.
+        transposed (bool): indicates the result tensor is transposed so that each thread holds consecutive elements in the same row instead of column, which is good for chained dot and global write.
         warps_per_cta (List[int]): Number of warps per CTA.
-        tiles_per_warp Optional(List[int]): Number of tiles per WARP. For mfma, every element is one.
-        elem_type Optional: Supported types are int32, fp32 and fp64. Default is fp32.
+        elem_type Optional(ttgl.dtype): Supported types are int32, fp32 and fp64. Default is fp32.
+        tiles_per_warp Optional(List[int]): Number of tiles per WARP. For mfma, if missing, use the default where we have unit tile size on all dimensions.
```

Comment:
Nit: This is mfma layout. Do we have a case not for mfma? So "For mfma," looks not useful at all. :)